### PR TITLE
Revert

### PR DESCRIPTION
**Pull request by rivalez**

### CODE REVIEW

**Comment by rivalez** [comment]
Just reverted last changes

File: battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketServer.java [hunk 1]
```diff
@@ -23,18 +21,18 @@ public boolean connect(String ip) {
...
-            logger.warning(e.getMessage());
+            e.printStackTrace();
         }
         return false;
     }
 
     @Override
     public void send(String allShips) {
-        logger.info(allShips);
+        System.out.println(allShips);
```
Comment (brrt2) [inline battleship-client/src/main/java/com/epam/solid/nie/client/communication/SocketServer.java]: I am not really sure why the logger is replaced by sout

File: battleship-client/src/main/java/com/epam/solid/nie/client/ui/Board.java [hunk 2]
```diff
@@ -11,20 +11,19 @@
...
  *
  */
 class Board extends Parent {
-    private Logger logger = Logger.getLogger(Board.class.getName());
     private static final int MAX_HEIGHT = 10;
     private static final int MAX_WIDTH = 10;
     private VBox rows = new VBox();
     private boolean enemy;
-    private List<Ship> allShips = new ArrayList<>();
+    List<Ship> allShips = new ArrayList<>();
```
Comment (brrt2) [inline battleship-client/src/main/java/com/epam/solid/nie/client/ui/Board.java]: I think replacing private by package-private usually does not bode well, though I do not know what impact it has on your code

File: battleship-client/src/main/java/com/epam/solid/nie/client/ui/Ship.java [hunk 3]
```diff
@@ -24,6 +24,7 @@ void hit(Point2D point2D) {
     }
 
     boolean isAlive() {
+        //return !(battleShip.getShipProperties().isEmpty());
```
Comment (brrt2) [inline battleship-client/src/main/java/com/epam/solid/nie/client/ui/Ship.java]: I do not think  comments should be left here

File: battleship-client/src/main/java/com/epam/solid/nie/client/ui/StartSceneController.java [hunk 4]
```diff
@@ -10,28 +10,26 @@
...
 
 import java.io.IOException;
-import java.util.logging.Logger;
 
 public class StartSceneController {
-    private Logger logger = Logger.getLogger(StartSceneController.class.getName());
 
     @FXML
-    private Button btnConnect;
+    private Button btn_connect;
```
Comment (brrt2) [inline battleship-client/src/main/java/com/epam/solid/nie/client/ui/StartSceneController.java]: I get the impression that this underscore separator breaks the naming convention of your code

**Comment by rivalez** [comment]
I know all comments are correct but I don't want mess up repository once again :P

**Pull request merged**